metadata-transformer/src/main/java/no/unit/nva/metadata/service/MetadataService.java

```diff
@@ -127,7 +166,6 @@ private JsonLdOptions avoidBlankNodeIdentifiersAndOmitDefaultsConfig() {
     private String getQueryAsString(URI uri) {
         var inputStream = inputStreamFromResources(MetadataService.QUERY_SPARQL);
         return new BufferedReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8)).lines()
-                .collect(Collectors.joining(NEWLINE_DELIMITER)).replaceAll(REPLACEMENT_MARKER, uri.toString());
+            .collect(Collectors.joining(NEWLINE_DELIMITER)).replaceAll(REPLACEMENT_MARKER, uri.toString());
```

Comment:
Should NEWLINE_DELIMITER be set to System.lineSeparator()?

Reply:
I believe so. Good find.